sparkle/src/components/DataTable.tsx

```diff
@@ -37,6 +38,7 @@ export function DataTable<TData extends TBaseData, TValue>({
   filter,
   filterColumn,
   initialColumnOrder,
+  mobileHiddenColumns = [],
```

Comment:
can we have something more flexible like 
`columnsBreakpoints={{ "lastUpdated": "sm" , "otherColumn": "md" }}` ?

Reply:
Sure, definitely better 👌🏼